### PR TITLE
scaffold single exhibition

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: single-exposiciones.php [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+    <div class="col col-6">
+
+      <ul class="filter-menu">
+      <?php
+        $exhibition_types = get_terms('tipo_de_exposicion');
+
+        if ($exhibition_types) {
+          foreach ($exhibition_types as $type) {
+            if ($post_terms[0]->ID === $type->ID) {
```
Comment (mixcas) [inline single-exposiciones.php]: this part of the code gives an error if `$post_terms` is empty

also is not working well
@varffs did u actually test this code? i think is `term_id` not `ID`…

Comment (varffs) [inline single-exposiciones.php]: yh i just wrote it blind

File: single-exposiciones.php [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      ?>
+        <li><a class="filter-term active" href="<?php echo get_term_link($type->term_id); ?>"><?php echo $type->name; ?></a></li>
+      <?php
+            } else {
+      ?>
+        <li><a href="<?php echo get_term_link($type->term_id); ?>"><?php echo $type->name; ?></a></li>
```
Comment (mixcas) [inline single-exposiciones.php:51]: this doesn't get the `filter-term` class?

Comment (varffs) [inline single-exposiciones.php:51]: it doesnt actually do anything without the active class?

Comment (mixcas) [inline single-exposiciones.php:51]: maybe not, i think this is bad practice tho

**Comment by mixcas** [comment]
`active` class stopped working on   ea51646

File: single-exposiciones.php [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+    $meta = get_post_meta($post->ID);
+    $post_terms = get_the_terms($post->ID, 'tipo_de_exposicion');
+
+    if (!empty($post_terms)) {
+      $post_term_id = $post_terms[0]->ID;
```
Comment (mixcas) [inline single-exposiciones.php]: @varffs `term_id` bro

**Pull request merged**